src/pages/RFOX/hooks/useStakingBalanceOfQuery.ts

```diff
@@ -18,18 +18,18 @@ type StakingBalanceOfQueryKey = ReadContractQueryKey<
 >
 type StakingBalanceOf = ReadContractReturnType<typeof erc20Abi, 'balanceOf', readonly [Address]>
 type UseStakingBalanceOfQueryProps<SelectData = StakingBalanceOf> = {
-  stakingAssetAccountAddress: string | undefined
+  accountId: AccountId | undefined
   stakingAssetId: AssetId | undefined
   select?: (stakingBalanceOf: StakingBalanceOf) => SelectData
   enabled?: boolean
 }
 const client = viemClientByNetworkId[arbitrum.id]
 
 export const getStakingBalanceOfQueryKey = ({
-  stakingAssetAccountAddress,
```

Comment:
Not necessarily a staking asset account address - can be a staking asset or user EOA address hence the generalization to `accountId`